src/services/textChanges.ts

```diff
@@ -483,107 +483,150 @@ namespace ts.textChanges {
             if (index < 0) {
                 return this;
             }
-            const end = after.getEnd();
             if (index !== containingList.length - 1) {
                 // any element except the last one
                 // use next sibling as an anchor
                 const nextToken = getTokenAtPosition(sourceFile, after.end, /*includeJsDocComment*/ false);
                 if (nextToken && isSeparator(after, nextToken)) {

```

Comment:
I don't think this actually changed, but does the insertion just silently fail if there's no next token or it's not a separator?